store/postgres/src/block_range.rs

```diff
@@ -17,6 +16,16 @@ pub type BlockNumber = i32;
 
 pub const BLOCK_NUMBER_MAX: BlockNumber = std::i32::MAX;
```

Comment:
This should now be `std::i32::MAX - 1` since `std::i32::MAX` is infinity.

Reply:
infinity is still `Unbounded`; I am happy to change this, but don't understand your reasoning. We use the constant to generate the same condition that is in `BLOCK_RANGE_CURRENT` for with `Store.get` and `Store.find`

Reply:
Yhea my comment didn't make much sense, this is fine.